docs/changes.rst

```diff
@@ -19,6 +19,29 @@ Released: not yet
 * Installation of this package using "setup.py" is no longer supported.
   Use "pip" instead.
 
+* Update to pywbem version 1.7.0 may cause issues with SSL because Python
+  packages may now require support of TLS version GE 1.2. Because the the
+  minimum version of the TLS is determined by the version of SSL actually
```

Comment:
On "version of the TLS is determined by the version of SSL". That is confusing if one interprets SSL as the earlier version of TLS. What you probably intend to talk about is the "version of the SSL library"

Reply:
Changed to say TLS protocol SSL library. DONE